src/Resources/Resources/ChangeLog.md

```diff
@@ -23,6 +23,7 @@
 ## Version 6.5.2
 * Fixed query issue when objectId in assignment is empty for `Get-DenyAssignment`
 * Fixed an issue where running deployment cmdlets with `-WhatIf` throws exception when formatting results with nested array changes
+* Updated `Publish-AzBicepModule` to include new optional parameter `-DocumentationUri`
```

Comment:
please move this line under line 21

Reply:
Updated here:
https://github.com/bhsubra/azure-powershell/commit/5061a4c031604b4c8dd8fcf5df08e4f72fdc3219

Thanks!